README.md

```diff
@@ -53,7 +53,9 @@ BuildKite | CMake         | Linux      | x86-64               | Tracing + Standa
 
 ## Architecture Overview
 
-![IREE Architecture](docs/website/docs/assets/images/iree_architecture.svg)
+<!-- TODO(scotttodd): switch to <picture> once better supported? https://github.blog/changelog/2022-05-19-specify-theme-context-for-images-in-markdown-beta/ -->
+![IREE Architecture](docs/website/docs/assets/images/iree_architecture_dark.svg#gh-dark-mode-only)
```

Comment:
note: this was exported from the same `iree_architecture.excalidraw.gz` file on https://excalidraw.com/, but with the dark theme option selected, and with the three stacked rectangles in the upper right changed from solid black to a lighter gray